README.md

```
@@ -1,23 +1,283 @@
-| `refactor` | 리팩토링 (기능 변화 없이 코드 구조 개선)     |
-| `test`     | 테스트 코드 추가 또는 수정                   |
+**꿈꾸고 싶은, 꿈꾸고 있는 50대 여성이 새로운 시작을 두드리는 곳 Do Dream!**
 
-## 🪃 브랜치
+### **_‘두드림’은 50대 전업주부가 흩어진 의지를 모아, 새로운 직업과 목표를 찾아 도전할 수 있도록 돕는 서비스 입니다._**
 
-feature → **feature/(이름)-#(이슈번호)**
+### 🔗 서비스 배포 주소 : https://www.dodream.site/
 
-bug → **bug/(이름)-#(이슈번호)**
+<img src="https://github.com/user-attachments/assets/88f83595-d4c5-4211-b8d4-f04d81504296">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)